google.html

```diff
@@ -0,0 +1,67 @@
+<!-- get default html settings-->
+<!DOCTYPE html>
+
+<html>
+
+<head>
```

Comment:
Each child should be indented 1 to it's parent.  Here <html> is the parent, so <head> should be indented one